src/tests/libs/widgets-loader.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, test } from 'vitest'
+
+import { WidgetType } from '@/types/widgets'
+
+test('Test widgets exist', async () => {
+  const loader = await Promise.allSettled(
+    Object.values(WidgetType).map((name) => import(`@/components/widgets/${name}.vue`))
+  )
+  const found_all_files = loader.every((file) => {
```

Comment:
We could maybe add the name of the missing files to inform the user, something like this

```suggestion
const missing_files = loader
    .map((file) => {
      if (file.status == 'fulfilled') {
        return null
      }

      const found = !file.reason.message.includes('Failed to load')

      return found ? null : file.reason.message
    })
    .filter((file) => file)

  const found_all_files = missing_files.length == 0

  let error_msg = ''
  if (!found_all_files) {
    error_msg = `\nFollowing widgets are missing: \n    - ${missing_files.join('\n    - ')}\n`
  }

  expect(found_all_files, error_msg).toBe(true)
```